indexing-service/src/test/java/org/apache/druid/indexing/common/task/KillUnusedSegmentsTaskTest.java

```diff
@@ -59,7 +59,7 @@ public void testKill() throws Exception
         newSegment(Intervals.of("2019-03-01/2019-04-01"), version),
         newSegment(Intervals.of("2019-04-01/2019-05-01"), version)
     );
-    final Set<DataSegment> announced = getMetadataStorageCoordinator().announceHistoricalSegments(segments);
+    final Set<DataSegment> announced = getMetadataStorageCoordinator().commitSegments(segments);
```

Comment:
It might be nice to rename `announced` to `committed`

Reply:
Sure 👍🏻 
If CI fails for any reason, I will include this change. Otherwise, I will include it in the other PR.